### ISSUE TITLE
[BUG] Missing Relationships

### ISSUE THREAD

**Issue by Crowguard** [ironmonk88/monks-enhanced-journal#468]
User upgraded MEJ from 10.6 to 10.9, and many of the relationships in journals are one sided and missing.
![image](https://user-images.githubusercontent.com/99844555/210857859-7f6d1dd3-7e95-43e3-9961-ed6689acff08.png)
(Ernsk's journal is linked in Anamaria's, but not vice versa. Image quality is what I was sent)

User reports these are the journal ID's in-Foundry. I can't find either ID in the JSON's relationships, but I may be missing something.

Anamaria: ZhApb9HYTDwgoXi0
Ernesk: y89qStbg5febrUlN

[Journals.zip](https://github.com/ironmonk88/monks-enhanced-journal/files/10354921/Journals.zip)

MEJ 10.6-10.9
Foundry 10.291

**Comment by Crowguard** [comment]
Update: User had a backup and reverted everything back to 10.6 and was able to see the relationships again. Then after upgrading to 10.9 a second time for testing, the relationships disappear again.

**Comment by ironmonk88** [comment]
The ID wouldn't be in the JSON 'cause Foundry strips out the id during an export.  Unfortunately any export will clear the relationships.
I checked the code, and the only change between 10.6 and 10.9 was to remove duplicate id's from the list.  Which only happens if a relationship was added incorrectly twice.  Different Journal entries shouldn't be affected.

I'm a bit at a loss.  I'd probably need to see a copy of the world to figure out what was happening.
